WordPress/src/main/java/org/wordpress/android/ui/reader/comments/ThreadedCommentsFragment.kt

```diff
@@ -0,0 +1,964 @@
+import org.wordpress.android.ui.reader.ReaderPostPagerActivity.DirectOperation.COMMENT_LIKE
+import org.wordpress.android.ui.reader.ReaderPostPagerActivity.DirectOperation.COMMENT_REPLY
+import org.wordpress.android.ui.reader.ReaderPostPagerActivity.DirectOperation.POST_LIKE
+import org.wordpress.android.ui.reader.actions.ReaderActions.CommentActionListener
+import org.wordpress.android.ui.reader.actions.ReaderActions.UpdateResult
+import org.wordpress.android.ui.reader.actions.ReaderCommentActions
+import org.wordpress.android.ui.reader.actions.ReaderPostActions
+import org.wordpress.android.ui.reader.adapters.ReaderCommentAdapter
+import org.wordpress.android.ui.reader.services.ReaderCommentService
+import org.wordpress.android.ui.reader.tracker.ReaderTracker
+import org.wordpress.android.ui.suggestion.Suggestion.Companion.fromUserSuggestions
+import org.wordpress.android.ui.suggestion.adapters.SuggestionAdapter
+import org.wordpress.android.ui.suggestion.service.SuggestionEvents.SuggestionNameListUpdated
+import org.wordpress.android.ui.suggestion.util.SuggestionServiceConnectionManager
+import org.wordpress.android.ui.suggestion.util.SuggestionUtils.setupUserSuggestions
+import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T.READER
+import org.wordpress.android.util.DisplayUtils
+import org.wordpress.android.util.EditTextUtils
+import org.wordpress.android.util.NetworkUtils
+import org.wordpress.android.util.ToastUtils
+import org.wordpress.android.util.ToastUtils.Duration.LONG
+import org.wordpress.android.util.WPActivityUtils
+import org.wordpress.android.util.WPSwipeToRefreshHelper
+import org.wordpress.android.util.analytics.AnalyticsUtils.AnalyticsCommentActionSource
+import org.wordpress.android.util.config.FollowByPushNotificationFeatureConfig
+import org.wordpress.android.util.helpers.SwipeToRefreshHelper
+import org.wordpress.android.util.redirectContextClickToLongPressListener
+import org.wordpress.android.viewmodel.observeEvent
+import org.wordpress.android.widgets.RecyclerItemDecoration
+import org.wordpress.android.widgets.WPSnackbar.Companion.make
+import java.util.Locale
+import javax.inject.Inject
+
+// TODO: as part of a refactor, this class was converted from the ReaderCommentListActivity.java to a kotlin
+// ThreadedCommentsActivity/ThreadedCommentsFragment pair. Remove SuppressWarnings exceptions below when
+// extracting logic from this fragment into the VM. Smaller warnings were fixed.
+@SuppressWarnings(
+        "EmptyFunctionBlock",
+        "LongMethod",
+        "LargeClass",
+        "NestedBlockDepth",
+        "ReturnCount",
+        "LongMethod",
+        "TooManyFunctions"
+)
+class ThreadedCommentsFragment : Fragment(R.layout.threaded_comments_fragment), OnConfirmListener, OnCollapseListener {
+    private var binding: ThreadedCommentsFragmentBinding? = null
+
+    private var mBlogId: Long = 0
```

Comment:
Since it's a kotlin file, should we remove the `m` prefix?

Reply:
I was initially keeping them with java convention trying to ease the comparison with original file for review, but since you already reviewed the code, makes sense 👍 . Changed here 7df770c6926b3403f7b946c7968acafc4bc7a520